src/auto/param_spec_expression.rs

```diff
@@ -0,0 +1,24 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::translate::*;
+use gobject;
+use gtk_sys;
+use std::fmt;
+
+glib_wrapper! {
+    pub struct ParamSpecExpression(Object<gtk_sys::GtkParamSpecExpression, ParamSpecExpressionClass>) @extends gobject::ParamSpec;
```

Comment:
This does not work like this and needs to be manual.

See https://github.com/gtk-rs/glib/blob/master/src/param_spec.rs for details. This also needs a `new()` function (`gtk_param_spec_expression()`) and inside GLib I should clean up things so that there *are* `new()` functions...

Reply:
Actually nevermind, it shouldn't have a `new()` function but basically work like https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/blob/master/gstreamer/src/param_spec.rs#L13 . That way it will work well with the subclass property API.

Reply:
I have disabled all the Gtk.*Expression types until they are handled properly

Reply:
Using Expressions from source code has been super weird so far, but thankfully you can use those from the UI file as well.

Reply:
https://github.com/gtk-rs/gtk4/issues/21